comienza_con_a.py

```diff
@@ -13,8 +12,6 @@
 entrada = entrada.upper()
```

Comment:
Está bien que cambies la entrada a mayúscula para facilitar las comparaciones en el proceso pero recuerda que necesitas la entrada original en el mensaje de salida. Así que podrías guardarla para usarla después:

    entrada_original = entrada

O, mejor aún, dejar la variable de entrada sin cambio y usar otra variable auxiliar en tu proceso:

    comparacion = entrada.upper()
    if comparacion[0] in ...